packages/services/src/services/Profile/Profile.js

```diff
@@ -59,6 +68,44 @@ class ProfileAPI {
         return { user: 'Unauthenticated' };
       });
   }
+
+  /**
+   * retrieve the cloud login status via cookie
+   *
+   *
+   * @returns {string} string determining login status
+   */
+  static checkCloudCookie() {
+    const cloudLogin = Cookies.get(_cookieName);
+
+    //console.log('Cloud Cookie', cloudLogin);
+
+    return { user: cloudLogin === '1' ? 'authenticated' : 'anonymous' };
```

Comment:
I think exporting the variable in the enum makes sense especially when it comes to the auth method types since we're using that in multiple packages, but for these cloud auth statuses, I think it's a bit overkill.